tests/foreman/cli/test_capsule.py

```diff
@@ -382,3 +386,25 @@ def test_positive_enable_bmc():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.skip_if_not_set('fake_capsules')
+@pytest.mark.tier1
+def test_positive_search_nondefault_proxy(request, target_sat):
+    """Search non-default proxy  with the home proxy
+
+    :id: caf51662-6b4e-11ed-baba-2b9d7b368002
+
+    :expectedresults: Non-default proxy can be searched
+
+    :BZ: 2077824
+
+    :customerscenario: true
+    """
+    proxy = _make_proxy(request, target_sat, options={'name': gen_string('alpha')})
+    path = target_sat.api.Capsule().path() + '?search=id+%21%3D+1'
+    response = client.get(path, auth=get_credentials(), verify=False)
```

Comment:
Does the verification need to be true here taking auth into account?

Reply:
for API requests we need false as we don't have ssl certs for sat